@here/harp-mapview/lib/Utils.ts

```diff
@@ -596,6 +596,62 @@ export namespace MapViewUtils {
         objectSize.gpuSize += attrBytes;
     }
 
+    function estimateGeometrySize(
+        geometry: THREE.Geometry | THREE.BufferGeometry,
+        objectSize: MemoryUsage,
+        visitedObjects: Map<string, boolean>
+    ): void {
+
+        const isNewObject =
+            geometry.uuid === undefined || visitedObjects.get(geometry.uuid) !== true;
+
+        if (isNewObject) {
```

Comment:
Use early return if you know there is nothing to do.

    if (!isNewObject) { return } 

Less indentation, prettier code